packages/material-ui-icons/src/16mp.js

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import createSvgIcon from './utils/createSvgIcon';
+
+export default createSvgIcon(
+  <path d="M13.5 9H15v1.5h-1.5zM19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-7 15.5h-1.5V14h-1v3H8v-3H7v4.5H5.5v-5c0-.55.45-1 1-1H11c.55 0 1 .45 1 1v5zm3.5 0H14v-6h3.5c.55 0 1 .45 1 1V16c0 .55-.45 1-1 1h-2v1.5zM10 5.5v6H8.5V7H7V5.5h3zm3 6c-.55 0-1-.45-1-1v-4c0-.55.45-1 1-1h3.5V7h-3v1h2c.55 0 1 .45 1 1v1.5c0 .55-.45 1-1 1H13zm2.5 2.5H17v1.5h-1.5z" />
```

Comment:
Not sure if relevant: https://material.io/resources/icons/?search=16m&icon=16mp&style=baseline

Reply:
Looks like a symbol for the number of MegaPixel. Why would we now include that?

Reply:
If the size of the package is still manageable with them, no objection. I want to raise that, soon or later, we will hit a bottleneck, and the best solution might be to remove all the low-value/usage icons. In Google Analytics, we track the usage of each icon, it could also help choose what to remove.

Reply:
Sure, so it's not relevant now since usage is not existing for new icons.

Reply:
Another leverage we can use is what font awesome supports. They have a track record for implementing the most requested icons.